### ISSUE TITLE
Reproduce performance

### ISSUE THREAD

**Issue by Iamal1** [ruotianluo/self-critical.pytorch#75]
Hi, @ruotianluo , thanks for your contribution. I met a problem I could not reproduce the TOPDOWN's result. With bu_data, XE training, I can get at most 1.120 Cider with many parameter settings I tried, (beam size, lr, batch_size,  more epochs) . 
And for RL, starting from 1.10 cider(xe 30epochs), at about 100epochs, I get Cider 1.201 but other metrics are still below the original. 
I'll appreciate a lot if you could give me some suggestions.

**Comment by minizon** [comment]
@Iamal1 Hi, did you get a stable cider boost when using SCST?  I tried multiple times and found that the cider value always got worse. I'll appreciate it if you could share your training policies.

**Comment by minizon** [comment]
I've achieved bleu-1 0.791, bleu-4 0.367, meteor 0.278, rouge_l 0.575, cider 1.22 (beam size 1) on the validation set using the following parameter setting:
self.caption_model              =  'topdown'
self.rnn_size                        =  1000
self.input_encoding_size  =  1000
self.att_feat_size              =  2048
self.att_hid_size                =  512
 ##  Optimization:  General
self.max_epochs  =  40
self.batch_size  =  100  
self.optim  =  'adam'
self.learning_rate  =  5e-4
self.learning_rate_decay_start  =  0
self.scheduled_sampling_start  =  0
self.self_critical_after  =  30
##  Evaluation/Checkpoint
self.save_checkpoint_every  =  1100
 self.val_images_use  =  -1
